plugins/processors/noise/README.md

```diff
@@ -0,0 +1,63 @@
+# Noise Processor
+
+The *Noise* processor is used to add noise to numerical field values. For each field a noise is generated using a defined probability densitiy function and added to the value. The function type can be configured as _Laplace_, _Gaussian_ or _Uniform_.
+Depending on the function, various parameters need to be configured:
+
```

Comment:
The readme needs a copy of the sample config. The requirement to have it in both places is kind of buried in the project developer docs. Here's a link: https://github.com/influxdata/telegraf/blob/master/docs/developers/REVIEWS.md#reviewing-plugin-code (second item in list)

Reply:
Thx, added it with the next commit.